config/routes.rb

```diff
@@ -17,6 +17,7 @@
   resources :customers
   resource :calendar, only: :show
   resources :appointments, only: %i(index show edit)
+  resources :holidays
```

Comment:
Can we start explicitly whitelisting the restful actions we support? It helps to understand what things we are supporting and where. Also cutting down the noise in `rails routes` is super helpful.